customization/dark-theme.md

```diff
@@ -120,7 +120,7 @@ The advantage of creating a dark themed Template VM is, that each AppVM which is
 
 1. Start VM
 
-    **Note:** In the case of App VM start the Template on which the AppVM is based on.
+    **Note:** Remember that if you want to make the change persistent, the change needs to be made in the TemplateVM, not the AppVM.
```

Comment:
The old wording seemed to me to be intentionally phrased to account for StandaloneVMs not having TemplateVMs. Your new wording does not account for that.